packages/commonwealth/server/workers/graphileWorker/tasks/captureGroupSnapshot.ts

```diff
@@ -0,0 +1,47 @@
+import { command, logger } from '@hicommonwealth/core';
+import { GroupSnapshot } from '@hicommonwealth/model';
+import { models } from '@hicommonwealth/model/db';
+import { systemActor } from '@hicommonwealth/model/middleware';
+import { GraphileTask, TaskPayloads } from '@hicommonwealth/model/services';
+
+const log = logger(import.meta);
+
+export const captureGroupSnapshotTask: GraphileTask<
+  typeof TaskPayloads.CaptureGroupSnapshot
+> = {
+  input: TaskPayloads.CaptureGroupSnapshot,
+  fn: captureGroupSnapshot,
+};
+
+export async function captureGroupSnapshot(payload: { groupId: number }) {
```

Comment:
I started placing these sync functions in libs/model so we can unit test them in isolation.. the graphile task can import from libs/model to wrap the job around it